cmd/crc/cmd/config/config.go

```diff
@@ -52,6 +53,8 @@ func RegisterSettings(cfg *config.Config) {
 
 	// Telemeter Configuration
 	cfg.AddSetting(ConsentTelemetry, "", config.ValidateYesNo, config.SuccessfullyApplied)
+
+	cfg.AddSetting(AutoLoginAs, "", config.ValidateAdminOrDeveloper, config.SuccessfullyApplied)
```

Comment:
I would probably even go as far as making developer the default,and always log in as this user after a succesful deployment...

... however, at times, the cluster does not reconcile in time to allow this to happen.

Reply:
My reason against this was to try and be less invasive with current behavior and so users don't get their context switched while working on a different one.
Like you're just running something on a different context and suddenly your `crc` is finally finished and takes over. 
Wanted it to be a conscious decision for now, but haven't got a that strong opinion on it.

Reply:
That makes sense... so the use is:

```
$ crc config set auto-login-as developer
```

and disable as:

```
$ crc config unset auto-login-as
```

which means until `unset` this would blow away the current one... and the unset if set this way can not be overriden from the `crc start` command as it would not take an empty value, right?

so this means that beforehand you would need to see this config is set, or at least as the final output that it happened and succeeded. so at least the user gets the feedback what happened. the current code does not have this output. this means the behaviour to the user can be unpredictable. especially in regards to the possible startup timeout this would make it unclear to the user if follow-up commands are actually running in the right context.

Reply:
Not sure if I get what you mean here.
This might be wrong, I just followed other flag implementations. It should be possible for the user to set login-as as a flag or in the config and disable it the same way (so it probably should accept an empty value on the flag to unset).

Regarding the output, I agree. Now that I know the login might not work instantly, it should write a message like "attempting to login" and either try silently until a timeout is reached or directly fail with an error afterwards.

Reply:
Attempting suggest very clearly this might fail. Uncertainty is not suggested in communicating.

Auto login ... [failed]

Would work better.  But yeah, a possible approach is to try this several times with a timeout. But Praveen might have an idea around this